codespell_lib/data/dictionary.txt

```diff
@@ -48509,6 +48516,7 @@ unepected->unexpected
 unepectedly->unexpectedly
 unequalities->inequalities
 unequality->inequality
+unequivalent->inequivalent
```

Comment:
I can find neither `unequivalent` nor `inequivalent` in these dictionaries:
* [Oxford Learner's](https://www.oxfordlearnersdictionaries.com/spellcheck/english/?q=inequivalent)
* [Cambridge](https://dictionary.cambridge.org/fr/dictionnaire/anglais/inequivalent)
* [Collins](https://www.collinsdictionary.com/spellcheck/english?q=inequivalent)
* [Merriam-Webster](https://www.merriam-webster.com/dictionary/inequivalent)
* [Longman](https://www.ldoceonline.com/spellcheck/english/?q=inequivalent)

But I do find `nonequivalent` in some of them:
* [Cambridge](https://dictionary.cambridge.org/fr/dictionnaire/anglais/nonequivalent)
* [Merriam-Webster](https://www.merriam-webster.com/dictionary/nonequivalent)

The Google Ngram viewer suggests `inequivalent` is more common in British English:
https://books.google.com/ngrams/graph?content=unequivalent%2Cinequivalent%2Cnonequivalent&year_start=1900&year_end=2019&corpus=en-2019

In any case, `unequivalent` is clearly to be avoided. I would suggest:
```suggestion
unequivalent->nonequivalent, inequivalent,
```

Reply:
never heard anyone use nonequivalent. inequivalent is much more common from my experience.

Reply:
@DimitriPapadopoulos I think this is ready to go unless you feel strongly about this one?

Reply:
@janosh I am not a native speaker. However:
* The Ngram Viewer shows that `nonequivalent` is in wider use than `inequivalent` in written English.
* Mainstream dictionaries contain `nonequivalent` bot not `inequivalent`.
* The Firefox spellchecker is showing `inequivalent` as a typo as I am writing this.
* The word [`inequivalent` is missing from _aspell_ dictionaries](http://app.aspell.net/lookup?dict=en_GB-large;words=inequivalant), which suggests `equivalent`, `nonequivalent` and `univalent` as possible fixes for `unequivalent`.

It might be worth [raising an issue against SCOWL](https://github.com/en-wl/wordlist/issues) itself, which is used as a reference by spellcheckers, instead of applying fixes to our typo dictionary that are inconsistent with SCOWL.

Reply:
But in the short term it's best to follow the above sources, isn't it?
```suggestion 
unequivalent->inequivalent
unequivalent->nonequivalent
```

Reply:
Feel free to make the change. I can't click commit suggestion and the github.dev editor is stuck in merge resolution for me.

Reply:
Indeed the GitHub editor is broken, it cannot edit files > 1 MB.